packages/components/src/Chips/InternalChipDismissible/InternalChipDismissible.tsx

```diff
@@ -3,33 +3,48 @@ import styles from "./InternalChipDismissible.css";
 import { useInternalChipDismissible } from "./hooks";
 import { InternalChipDismissibleInput } from "./InternalChipDismissibleInput";
 import { InternalChipDismissibleProps } from "./InternalChipDismissibleTypes";
-import { ChipDismissible } from "../ChipDismissible";
+import { InternalChip } from "../InternalChip";
+import { InternalChipButton } from "../InternalChipButton";
 
 export function InternalChipDismissible(props: InternalChipDismissibleProps) {
   const {
     availableChipOptions,
     ref: wrapperRef,
     sortedVisibleChipOptions,
-    handleChipClick,
     handleChipAdd,
+    handleChipClick,
+    handleChipKeyDown,
     handleChipRemove,
     handleCustomAdd,
     handleEmptyBackspace,
+    handleWrapperKeyDown,
   } = useInternalChipDismissible(props);
 
   return (
     <div
       ref={wrapperRef}
       className={styles.wrapper}
       data-testid="dismissible-chips"
+      onKeyDown={handleWrapperKeyDown}
     >
       {sortedVisibleChipOptions.map(chip => (
-        <ChipDismissible
+        <InternalChip
           key={chip.value}
           {...chip}
+          onKeyDown={handleChipKeyDown(chip.value)}
           onClick={handleChipClick(chip.value)}
-          onRequestRemove={handleChipRemove(chip)}
-        />
+          suffix={
+            <InternalChipButton
+              icon="remove"
+              {...chip}
+              onClick={handleChipRemove(chip.value)}
+            />
+          }
+        >
+          <span className={styles.visuallyHidden}>
+            Press delete to remove chip
+          </span>
```

Comment:
This actually gets read separately on voice over. Basically it'll go
- 🔈  Mando Press delete to remove this chip. Group.
- _User press right arrow key_
- 🔈 Press delete to remove this chip
- _User press right arrow key_
- 🔈 End of group
- _User press right arrow key_
- _Finally goes to the next chip_

I think instead of passing in a children, we should pass in a computed `aria-label` where it's the chip label + "press delete" thing. That way we don't need this extra element and it's styling and it's just a string that gets added as an html attribute.

Reply:
Updated here: https://github.com/GetJobber/atlantis/pull/872/commits/b2db210eb745cc816719f2794edb13b7322ac895, hope that works :)